### DISCUSSION TITLE
Add cerebras and groq support

### DISCUSSION THREAD

**Discussion by threatthriver** [cline/cline#1005] [Feature Requests]
If we have access to the groq and cerebras so we can use this for free and with more limit and with different models

**Comment by mausch** [comment]
groq is OpenAI-compatible already, no need to support it explicitly https://console.groq.com/docs/openai

**Comment by edwin-truthsearch-io** [comment]
Cerebras is there, and there is a PR for Groq #3898. Please upvote!
